resources/translations/en.edn

```diff
@@ -170,15 +170,15 @@
          :apply "Apply"
          :apply-for-bundle "Apply for %1 resources as a bundle "
          :header "%1 resources in the cart"
-         :intro [:div.intro [:p "Add resources to the cart to apply for them. If several resources can be applied to with the same application, they will be automatically grouped into a bundle."]]
+         :intro [:div.intro [:p "Apply for access by adding resources to the cart. When applying for multiple rights at the same time, resources using the same application process are bundled together as one application."]]
          :remove "Remove"}
   :catalogue {:catalogue "Catalogue"
               :header "Resource"
-              :intro [:p "The catalogue contains all the resources that you can apply for in this system."]
+              :intro [:p "Access can be applied for the following resources."]
```

Comment:
Hm, I'd like to add a colon to the end of the sentence, but the problem is that the list of resources does not follow.

I think I actually prefer the old version, it is feels more general and more descriptive.

Reply:
The old version is partly redundant and doesn't sound like completely proper English.

Reply:
I'm fine with how it ended up.